### PR TITLE
Update contribution guidelines

### PR DESCRIPTION
**Pull request by tnecnivkcots**
merging the changes made on the contribution guidelines (seen [here](https://github.com/pyiron/pyiron/pull/1410)

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -322,69 +324,100 @@ The build status for pyiron and all sub packages are given below
...
+In order to keep the tags consistent please follow the `Git-Tag-Guide`_.
 The tag format consists of a tag_prefix (<package name>-) and the release version, for example::
+     pyiron-0.2.0
+2. Automatically create PyPi package
+  After the tag is created, the `Deploy-Workflow`_ is triggered, which creates the PyPi Package.
+  The configuration of the release is included in the `setup.ctg file`_.
+  This Workflow first installs all dependencies, then allows for future versions of the dependencies and builds the package. After that the package is published to `pip`_.
+3. Automatically create conda-forge package
+  This release than is recognized by a conda-forge bot, which triggers a new pull request for the conda-forge package and merges automatically if all tests pass.
+  These tests are defined as `GitHub-Action-Workflows`_ and are triggered for every new pull request. More information can be found in the next chapter.
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:360]: The tests on conda-forge are not our unit tests.
```suggestion
```
Probably, better add this as preparation step: 
```
Check if all tests as explained in the next chapter have passed
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -322,69 +324,100 @@ The build status for pyiron and all sub packages are given below
...
+4. Docker images
+  The docker images are maintained manually and therefore not updated with every release. The docker images are build using hte conda packages and can be found in different variants under https://github.com/pyiron/docker-stacks
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:362]:
```suggestion
  The docker images are maintained manually and therefore not updated with every release. The docker images are build using the conda packages and can be found in different variants under https://github.com/pyiron/docker-stacks
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -322,69 +324,100 @@ The build status for pyiron and all sub packages are given below
...
+**deploy.yml**
 
-Those pre-release versions are named `<version_number>.post0.dev<release number>` ::
+This workflow is used to upload and deploy a new release to PyPi. 
+First, the install dependencies in order to create the PyPi distribution.
+After that, the version restriction of the dependencies are lifted to allow for future versions and the PyPi package is build according to the setup.py
+This release is then uploaded to PyPi, but only if it is tagged correctly.
 
-   0.2.0.post0.dev1
+**docs.yml*
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:403]:
```suggestion
**docs.yml**
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -419,17 +452,17 @@ This should output:
 If you see your job class in the list, proceed to step 3. If not, 
 
 
-**2. Check if the job class in initialized in ``__init__.py`` of the module**
+**2. Check if the job class is initialized in ``__init__.py`` of the module**
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:455]:
```suggestion
**2. Check if the job class is initialized in `__init__.py` of the module**
```

**Review by niklassiemer — COMMENTED** [review]
Small nits, otherwise LGTM

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -69,8 +71,8 @@ What should I know before I get started?
 pyiron developer meetings
 -------------------------
 If you are interested in discussing pyiron's development, we encourage you to virtually
-participate in the weekly pyiron developer meeting at 14:00 german time (GMT+2).
-Check the discussion page for details.
+participate in the pyiron developer meeting Mondays at 15:00 german time (GMT+2).
+Contact us if you want to participate in the weekly virtual pyiron developer meeting and check the discussion page for details.
```
[the remark this answers is not in the document]
Comment (tnecnivkcots) [reply CONTRIBUTING.rst:75]: Oh, then this information was already before not usefull.
Thank you for the change.

**Pull request merged**